main/CMakeLists.txt

```diff
@@ -128,7 +125,7 @@ if (MINGW)
    endforeach (QtLibrary ${QT_LIBRARIES})
    list(REMOVE_DUPLICATES QtInstallLibraries)
 
-   target_link_libraries(mscore ${QT_LIBRARIES})
+   # target_link_libraries(mscore ${QT_LIBRARIES})
```

Comment:
Isn't this line needed to link the executable file successfully?

Reply:
Apparently not, on the contrary, it prevents to link the executable file successfully

Reply:
Actually this one line is the core of this fix, it alone fixes the MinGW builds

Reply:
I think the Qt dependencies are already included in library libmscoreapp.a. CMake does some of its magic to determine the linking order ( https://cmake.org/cmake/help/v2.8.9/cmake.html#command%3atarget_link_libraries ) and apparently inserting a direct dependency of Qt libraries to mscore target brings the Qt libraries at the beginning of the linking order, just after libmscoreapp.a and this gives problem to the other .a libraries which are listed afterwards.